health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/SetupCampaign.js

```
@@ -65,21 +65,32 @@ const SetupCampaign = ({ hierarchyType, hierarchyData }) => {
+  //       name: "HierarchySchema",
+  //       filter: `[?(@.type=='${window?.Digit?.Utils?.campaign?.getModuleName()}')]`,
+  //     },
+  //   ],
+  //   { select: (MdmsRes) => MdmsRes },
+  //   { schemaCode: `${CONSOLE_MDMS_MODULENAME}.HierarchySchema` }
+  // );
+
+  const { 
+    data: BOUNDARY_HIERARCHY_TYPE, 
+    isLoading: hierarchyLoading,
+    rawData: rawData
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)